packages/web-components/fast-components/src/avatar/index.ts

```diff
@@ -20,7 +20,7 @@ export class Avatar extends FoundationAvatar {
      * HTML Attribute: src
      */
     @attr({ attribute: "src" })
-    public imgSrc: string;
+    public imgSrc = "";
```

Comment:
Any reason to not keep the typings, especially initialized, they won't be null or undefined?
```suggestion
    public imgSrc: string = "";
```

Reply:
I was unsure on what y'all prefer on this one (I've been at companies that prefer either one). Happy to put it back 😄